doc/source/whatsnew/v0.22.0.txt

```diff
@@ -45,7 +45,7 @@ Other API Changes
 - :class:`Timestamp` will no longer silently ignore unused or invalid `tz` or `tzinfo` keyword arguments (:issue:`17690`)
 - :class:`Timestamp` will no longer silently ignore invalid `freq` arguments (:issue:`5168`)
 - :class:`CacheableOffset` and :class:`WeekDay` are no longer available in the `tseries.offsets` module (:issue:`17830`)
-
+- :function:`tseries.frequencies.get_freq_group` and :list:`tseries.frequencies.DAYS` can now be accessed in `_libs.tslibs.resolution` (:issue:`18034`)
```

Comment:
I would just say they are removed from the private API, no need to point to a private file